homeassistant/components/tessie/binary_sensor.py

```diff
@@ -37,11 +37,6 @@ class TessieBinarySensorEntityDescription(BinarySensorEntityDescription):
         device_class=BinarySensorDeviceClass.HEAT,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
-    TessieBinarySensorEntityDescription(
```

Comment:
Please don't make a breaking change instead:
- disable the old entity by default
- start deprecation period 
- create repair issue when the old entity is used in automations and scripts

After the deprecation period, please remove in a follow-up the sensor

Reply:
Breaking change removed, just need to figure out how to work in the disabled by default to `BinarySensorEntityDescription`

Reply:
Ah yes, `entity_registry_enabled_default`. I think I might not remove it, but just leave it disabled. As Joostlek points out it may still be useful for automation.